### PR TITLE
Updated the new pokemon tutorial for 1.10

### PR DESCRIPTION
**Pull request by hedara90**
## Description

What it says on the tin.

## Things to note in the release changelog:

- Some changes compared to previous.

## **Discord contact info**

hedara

### CODE REVIEW

File: docs/tutorials/how_to_new_pokemon_1_10_0.md [hunk 1]
```diff
@@ -0,0 +1,1153 @@
...
+    const u8 gMonIcon_Pecharunt[] = INCBIN_U8("graphics/pokemon/pecharunt/icon.4bpp");
+#if P_FOOTPRINTS
+    const u8 gMonFootprint_Pecharunt[] = INCBIN_U8("graphics/pokemon/pecharunt/footprint.1bpp");
+#endif //P_FOOTPRINTS
+#if OW_POKEMON_OBJECT_EVENTS
+    const u32 gObjectEventPic_Pecharunt[] = INCBIN_COMP("graphics/pokemon/pecharunt/overworld.4bpp");
+#if OW_PKMN_OBJECTS_SHARE_PALETTES == FALSE
+    const u32 gOverworldPalette_Pecharunt[] = INCBIN_U32("graphics/pokemon/pecharunt/overworld_normal.gbapal.lz");
+    const u32 gShinyOverworldPalette_Pecharunt[] = INCBIN_U32("graphics/pokemon/pecharunt/overworld_shiny.gbapal.lz");
+#endif //P_FAMILY_PECHARUNT
```
Comment (Pawkkie) [inline docs/tutorials/how_to_new_pokemon_1_10_0.md:515]:
```suggestion
    const u32 gShinyOverworldPalette_Pecharunt[] = INCBIN_U32("graphics/pokemon/pecharunt/overworld_shiny.gbapal.lz");
#endif //OW_PKMN_OBJECTS_SHARE_PALETTES
#endif //OW_POKEMON_OBJECT_EVENTS
#endif //P_FAMILY_PECHARUNT

    const u32 gMonFrontPic_Egg[] = INCBIN_U32("graphics/pokemon/egg/anim_front.4bpp.lz");
    const u32 gMonPalette_Egg[] = INCBIN_U32("graphics/pokemon/egg/normal.gbapal.lz");
    const u8 gMonIcon_Egg[] = INCBIN_U8("graphics/pokemon/egg/icon.4bpp");
```

File: docs/tutorials/how_to_new_pokemon_1_10_0.md [hunk 2]
```diff
@@ -0,0 +1,1153 @@
...
+ const struct SpeciesInfo gSpeciesInfo[] =
+ {
+     ...
+     [SPECIES_MEWTHREE] =
+     {
+        ...
+        .abilities = { ABILITY_INSOMNIA, ABILITY_NONE, ABILITY_NONE },
+        .bodyColor = BODY_COLOR_PURPLE,
++       .isLegendary = TRUE,
++       .parfectIVCount = LEGENDARY_PERFECT_IV_COUNT,
```
Comment (Pawkkie) [inline docs/tutorials/how_to_new_pokemon_1_10_0.md:666]:
```suggestion
+ .perfectIVCount = LEGENDARY_PERFECT_IV_COUNT,
```

File: docs/tutorials/how_to_new_pokemon_1_10_0.md [hunk 3]
```diff
@@ -0,0 +1,1153 @@
...
+```diff
+ const struct SpeciesInfo gSpeciesInfo[] =
+ {
+     ...
+     [SPECIES_MEWTHREE] =
+     {
+        ...
+        .abilities = { ABILITY_INSOMNIA, ABILITY_NONE, ABILITY_NONE },
+        .bodyColor = BODY_COLOR_PURPLE,
++       .isLegendary = TRUE,
```
Comment (Pawkkie) [inline docs/tutorials/how_to_new_pokemon_1_10_0.md:665]: None of these flags are listed in the example template in `species_info.h`, not sure whether or not we want to add them (it seems they haven't been added in the past, figured it's worth considering whether we want to continue to leave them out)

**Review by Pawkkie — COMMENTED** [review]
Minor typos / updates to code involved in codeblocks, still reads great!

File: docs/tutorials/how_to_new_pokemon_1_10_0.md [hunk 4]
```diff
@@ -0,0 +1,1153 @@
...
+     ...
+     [SPECIES_MEWTHREE] =
+     {
+        ...
+        .pokemonScale = 256,
+        .pokemonOffset = 0,
+        .trainerScale = 290,
+        .trainerOffset = 2,
++       .frontPic = gMonFrontPic_Mewthree,
++       .frontPicSize = MON_COORDS_SIZE(64, 64),
```
Comment (Pawkkie) [inline docs/tutorials/how_to_new_pokemon_1_10_0.md:587]: This doesn't match the structure of the rest of these anymore, which are all in the form
```
        .frontPicSize = P_GBA_STYLE_SPECIES_GFX ? MON_COORDS_SIZE(56, 56) : MON_COORDS_SIZE(64, 48),
        .frontPicYOffset = P_GBA_STYLE_SPECIES_GFX ? 4 : 9,
        .backPicSize = P_GBA_STYLE_SPECIES_GFX ? MON_COORDS_SIZE(64, 48) : MON_COORDS_SIZE(56, 64),
        .backPicYOffset = P_GBA_STYLE_SPECIES_GFX ? 10 : 3,
```
what this means is probably worth clarifying

File: docs/tutorials/how_to_new_pokemon_1_10_0.md [hunk 5]
```diff
@@ -0,0 +1,1153 @@
+<!--1.9.x-->
```
Comment (Pawkkie) [inline docs/tutorials/how_to_new_pokemon_1_10_0.md:1]:
```suggestion
<!--1.10.x-->
```

File: docs/tutorials/how_to_new_pokemon_1_10_0.md [hunk 6]
```diff
@@ -0,0 +1,1153 @@
...
+This is a modified version of [the original tutorial about adding new Pokémon species available in Pokeemerald's wiki](https://github.com/pret/pokeemerald/wiki/How-to-add-a-new-Pokémon-species).
+
+Despite the persistent rumors about an incredibly strong third form of Mew hiding somewhere, it actually wasn't possible to catch it... OR WAS IT?
+In this tutorial, we will add a new Pokémon species to the game.
+
+## IMPORTANT: This tutorial applies to 1.9.x versions.
```
Comment (Pawkkie) [inline docs/tutorials/how_to_new_pokemon_1_10_0.md:7]:
```suggestion
## IMPORTANT: This tutorial applies to 1.10.x versions.
- [Version 1.9.x](how_to_new_pokemon_1_9_0.md)
```

**Review by Pawkkie — COMMENTED** [review]
Sorry clicked submit early

File: docs/tutorials/how_to_new_pokemon_1_10_0.md (hunk 4 above)
Comment (hedara90) [reply docs/tutorials/how_to_new_pokemon_1_10_0.md:588]: It is clarified further down, and it wouldn't be relevant to a custom pokemon which doesn't have an original gen 3 sprite

File: docs/tutorials/how_to_new_pokemon_1_10_0.md (hunk 3 above)
Comment (hedara90) [reply docs/tutorials/how_to_new_pokemon_1_10_0.md:666]: Their purpose is listed further down, and the example is specifying an evolution to Mew that's legendary, so either the example should change, or this stays.

Comment (Pawkkie) [reply docs/tutorials/how_to_new_pokemon_1_10_0.md:673]: I don' think anything in the document needs to change, what I meant is that if it were me I'd probably add all of these flags as `.isLegendary =FALSE` etc. at the bottom of the example in `species_info.h` so users have all of the options available when they copy paste the template. I don't think anything about the guide itself needs to change, I mean adding the parameters to the `species_info.h` template. I'm okay to leave it as is if you disagree though :)

File: docs/tutorials/how_to_new_pokemon_1_10_0.md (hunk 4 above)
Comment (Pawkkie) [reply docs/tutorials/how_to_new_pokemon_1_10_0.md:595]: I am missing the clarification on `P_GBA_STYLE_SPECIES_GFX `, but if this isn't relevant to custom graphics anyways I'm okay with it being ignored :)

**Comment by hedara90** [comment]
Everything resolved, pending other opinions.

**Review by Pawkkie — APPROVED** [review]
Approving, if you want to wait for a greater consensus feel free!

**Comment by pkmnsnfrn** [comment]
I skimmed, lgtm, won't merge because I'd like one more pair of eyes who has actually added species in the past

**Comment by hedara90** [comment]
Is this where we call in someone unrelated that tries the instructions?

**Review by Pawkkie — APPROVED** [review]

**Pull request merged**